docker-compose.yml

```diff
@@ -79,10 +79,13 @@ services:
   #   container_name: gateway_sumologic
   #   volumes:
   #     - /var/run/docker.sock:/var/run/docker.sock
+  #     - ~/platform/custom-sumo-sources.json:/etc/custom-sumo-sources.json
   #   environment:
-  #     - SUMO_ACCESS_ID=[insert API key here]
+  #     - SUMO_ACCESS_ID=[insert API ID here]
   #     - SUMO_ACCESS_KEY=[insert API key here]
   #     - SUMO_COLLECTOR_NAME=[insert name here]
+  #     - SUMO_COLLECTOR_NAME_PREFIX=GCIS Data Gateway-
```

Comment:
It's ok to add space but I just feel like adding spaces may cause confusion of users. Since the prefix is already changed using GCIS instead of platform, do we keep the old format?

Reply:
Also, I just tried if I add space at the end, the space will be ignore. For example,
SUMO_COLLECTOR_NAME_PREFIX=GCIS Data Gateway - (with space in the end)
SUMO_COLLECTOR_NAME=Sandbox Test

The collector name shown in Sumologic will be "GCIS Data Gateway -Sandbox Test" without space between "-" and "Sandbox". 

If I add double quotation:
SUMO_COLLECTOR_NAME_PREFIX="GCIS Data Gateway - "

The final collector name shown in Sumologic will be GCIS Data Gateway - "Sandbox Test which is weird. There is a space and a double quotation between "-" and "Sandbox". Looks like Sumologic doesn't handle it as we expected.

Probably keep it without space will make it simple